### ISSUE TITLE
manifest-tool can't merge images with buildit attestation enabled: Cannot include an image in a manifest list/index which is already a multi-platform image

### ISSUE THREAD

**Issue by mateuszkwiatkowski** [estesp/manifest-tool#233]
This is continuation of https://github.com/estesp/manifest-tool/issues/199. Displaying manifests with attestation records properly was fixed but manifest-tool still can't merge such images and throws an error msg:

```
msg="Cannot include an image in a manifest list/index which is already a multi-platform image: git.mycompany.com:5001/mycompany/project:build-aarch64-3-amd64"
```

As the next step manifest-tool needs to be able to merge such images. As a workaround one can disable BuildKit's attestation by passing `--provenance=false` argument to `docker buildx`:

```docker buildx build  --provenance=false --pull --push  [...]```

**Comment by estesp** [comment]
I have fixed this in the development (main) branch, but there is no release that includes it yet: https://github.com/estesp/manifest-tool/commit/11b168ba94a9c23a1d0627bc0df279dc80c01c24

I guess it's time to do a release!

**Comment by mateuszkwiatkowski** [comment]
@estesp there's another important fix that mandates release: https://github.com/estesp/manifest-tool/issues/223

Can't wait to see the new version! :-)

**Comment by estesp** [comment]
Solved in #218 and now released in v2.1.0 today. Thanks!
